draft/examples/reconciliation-query-batch/invalid/multiple-types.json

```diff
@@ -1,8 +1,15 @@
 {
   "queries": [
     {
-      "query": "Christel Hanewinckel",
-      "type": ["DifferentiatedPerson", "FictionalCharacter"],
+      "properties": [
```

Comment:
The examples above use `props` not `properties`. Are both accepted or is one of them wrong?

Reply:
I don't think we should accept both.  `properties` should be the only one, in my opinion to avoid confusion.

Reply:
Only "properties" is accepted. If you are referring to [draft/examples/reconciliation-query-batch/invalid/misnamed-property.json](https://github.com/reconciliation-api/specs/pull/149/files/399d613ac0d6eb7b647e500121d5c494c15ed0b9#diff-902a970609dca55e649281df9fb19e1fdb52c2520b70d222702bf0813fc872c5), then it is intentional that this JSON is wrong (see the `invalid/misnamed-property` part of the path - it is a negative example in our test suite). If you are aware of any other place where we use `props`, it's likely a problem that needs fixing indeed.

Reply:
Sorry, I missed that was an error case.